spec/components/filter_component_spec.rb

```diff
@@ -76,7 +76,19 @@
     expect(result.css('#status-offer_withdrawn').attr('checked')).to eq(nil)
   end
 
-  it 'when filters have been selected filters dialogue to appear' do
+  it 'when filters have been selected filters hidden text to appear' do
```

Comment:
```suggestion
  it 'when filters have been selected hidden text is displayed' do
```
Using the word filters here makes it a bit difficult ot understand this example.